db/migrate/20230725143247_add_confirmed_at_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddConfirmedAtToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :confirmed_at, :datetime
+  end
+end
```

Comment:
- **[OPTIONAL]** This file causes migration errors when migrating the db. You already have `confirmed_at` in the users table. This file is not needed. Kindly remove it.